.travis.yml

```diff
@@ -76,12 +76,8 @@ script:
    #       error "DISTCHECK (WITH NEWTHEORY TESTS) FAILED";
    #   }
    makeCheck() {
-     cd build
      make -j2 check ARGS='-LE regress[1-4]' CVC4_REGRESSION_ARGS='--no-early-exit' || error "BUILD/UNIT/SYSTEM/REGRESSION TEST FAILED"
-   }
-   makeExamples() {
-     cd build
-     make -j2 examples || error "COULD NOT BUILD EXAMPLES${normal}";
+     ctest -j2 -R examples/ || error "RUNNING EXAMPLES FAILED"
```

Comment:
`-L example` should also work here.

Reply:
Yes, and it's also the cleaner way to do this here.

Reply:
Yup, that makes sense :)